src/scripts/timezone.js

```diff
@@ -29,7 +30,27 @@ const TIMEZONES = {
   pst: "America/Los_Angeles",
 };
 
-const matcher = /(\d{1,2}:\d{2}\s?(am|pm)?)\s?(((ak|a|c|e|m|p)(s|d)?t)|:(eastern|central|mountain|pacific)-time-zone:)?/i;
+const timeString = (zonedDateTime, ampm) => {
+  const { hour: h, minute } = Temporal.PlainTime.from(zonedDateTime);
+
+  let ampmStr = "";
+  if (ampm) {
+    if (h < 12) {
+      ampmStr = " am";
+    } else {
+      ampmStr = " pm";
+    }
+  }
+
+  const hour = h === 0 ? 12 : h;
+
+  return `${hour > 12 ? hour - 12 : hour}:${
+    minute > 10 ? "" : "0"
+  }${minute}${ampmStr}`;
+};
```

Comment:
Is there a way to use `Intl.DateTimeFormat` to implement this?

If not, and we don't want to keep moment, what about considering a smaller library like [d3-time-format](https://github.com/d3/d3-time-format) around?